viewer/src/main/webapp/viewer-html/components/EditBulk.js

```diff
@@ -0,0 +1,396 @@
+            attributeNames.forEach(function (attributeName) {
+                var input = me.inputContainer.getForm().findField(attributeName);
+                me.originalFormProperties[attributeName] = {
+                    allowBlank: input.allowBlank,
+                    emptyText: input.emptyText
+                }
+            });
+        },
+        initAttributeInputs: function (appLayer) {
+            viewer.components.EditBulk.superclass.initAttributeInputs.call(this, appLayer);
+            this.storeOriginalFormProperties();
+            this.selectedFeatures = new SelectedFeatures(
+                this.config.viewerController.mapComponent.getMap(),
+                this.getListOfAttributeNamesThatAreMappedToFormFields()
+            );
+        },
+        makeFormFieldMultiple: function (attributeName) {
+            var field = this.inputContainer.getForm().findField(attributeName);
+            var placeholder = this.buildPlaceholderForAttributeWithMultipleValues(attributeName);
+            field.setEmptyText(placeholder);
+            field.allowBlank = true;
+            field.setValue('');
+            field.validate();
+        },
+        buildPlaceholderForAttributeWithMultipleValues: function (attributeName) {
+            var LEEG = '<leeg>';
+            var MEERDERE_WAARDES = 'Meerdere';
+            var values = this.selectedFeatures.gatherUniqueValuesForAttributeFromSelectedFeatures(attributeName);
+            values = values.map(function (value) {
+                if (typeof value === 'undefined' || value === null) {
+                    return LEEG
+                } else {
+                    return '"' + value + '"';
+                }
+            });
+            return MEERDERE_WAARDES + ' (' + values.join(', ') + ')';
+        },
+        makeFormFieldScalar: function (attributeName) {
+            var field = this.inputContainer.getForm().findField(attributeName);
+            var originalProperties = this.originalFormProperties[attributeName];
+            field.setEmptyText(originalProperties.emptyText);
+            field.allowBlank = originalProperties.allowBlank;
+            field.validate();
+        },
+        edit: function () {
+            this.selectedFeatures.deselectAll();
+            return viewer.components.EditBulk.superclass.edit.call(this);
+        },
+        mapClicked: function (toolMapClick, comp) {
+            if (this.mode === null) {
+                return;
+            }
+            this.showMobilePopup();
+            if (this.mode === "new") {
+                return;
+            }
+            this.getContentContainer().mask(i18next.t('viewer_components_edit_16'));
+            var coords = comp.coord;
+            this.lastClickCoords = coords;
+            this.getFeaturesForCoords(coords);
+        },
+        getFeaturesForCoords: function (coords) {
+            return viewer.components.EditBulk.superclass.getFeaturesForCoords.call(this, coords);
```

Comment:
If the methodname is the same, I think you can omit it here: it will get automatically called because of the Ext inheritance.